x-pack/plugins/ml/server/models/job_validation/messages.json

```diff
@@ -35,57 +35,70 @@
   },
   "categorization_filters_valid": {
     "status": "SUCCESS",
-    "text": "Categorization filters"
+    "text": "Categorization filters checks passed.",
+    "url": "https://www.elastic.co/guide/en/x-pack/{{version}}/ml-configuring-categories.html#ml-configuring-categories"
```

Comment:
I think this should point to this URL instead (with {{version}} as appropriate): https://www.elastic.co/guide/en/elastic-stack-overview/current/ml-configuring-categories.html

Reply:
With respect to: 
> "Cardinality of over_field \"{{fieldName}}\" is below 10 and might be less suitable for population analysis."

You might consider changing this to ... "might not be suitable".  Otherwise it makes me think "less suitable than what?"